packages/wrangler/src/vectorize/insert.ts

```diff
@@ -50,10 +57,17 @@ export async function handler(
 	const config = readConfig(args.config, args);
 	const rl = createInterface({ input: createReadStream(args.file) });
 
-	if (Number(args.batchSize) > VECTORIZE_MAX_BATCH_SIZE) {
+	if (args.deprecatedV1 && Number(args.batchSize) > VECTORIZE_MAX_BATCH_SIZE) {
 		logger.error(
 			`🚨 Vectorize currently limits upload batches to ${VECTORIZE_MAX_BATCH_SIZE} records at a time.`
 		);
+	} else if (
+		!args.deprecatedV1 &&
+		Number(args.batchSize) > VECTORIZE_V2_MAX_BATCH_SIZE
+	) {
```

Comment:
let's put a warning for V2 with batch size > 1 and < 5000:
`The global rate limit for the Cloudflare API of 1200 requests per five minutes. Please use a maximum of 5000 vectors per embeddings.ndjson file to stay within the service limits`

Reply:
Updated!